### PR TITLE
Fix LJ metrics naming

### LINKED ISSUE #390: Fix metrics naming (again)
lj_gc_allocated counter metrics should have "_total" suffix
lj_gc_freed counter metrics should have "_total" suffix
lj_gc_steps_atomic counter metrics should have "_total" suffix
lj_gc_steps_finalize counter metrics should have "_total" suffix
lj_gc_steps_pause counter metrics should have "_total" suffix
lj_gc_steps_propagate counter metrics should have "_total" suffix
lj_gc_steps_sweep counter metrics should have "_total" suffix
lj_gc_steps_sweepstring counter metrics should have "_total" suffix
lj_jit_snap_restore counter metrics should have "_total" suffix
lj_jit_trace_abort counter metrics should have "_total" suffix
lj_strhash_hit counter metrics should have "_total" suffix
lj_strhash_miss counter metrics should have "_total" suffix

Steps to fix:
1) Add duplicate metric for each of mentioned above (with different name)
2) Add log message about deprecation

### PR DESCRIPTION
**Pull request by panticonur**
Fix LJ metrics naming - add `_total` suffix.

I didn't forget about

- [x] Tests
- [x] Changelog

Close #390

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - label `thread` for per thread reporting net statistics metrics.
 
+### Deprecated
```
Comment (yngvar-antonsson) [inline CHANGELOG.md:18]: Please add section "Added" with new metrics
And log message like this: https://github.com/tarantool/metrics/blob/c3aeda62bbeb12f6b4506620d635bada742af6ae/metrics/init.lua#L129

File: metrics/tarantool/luajit.lua [hunk 2]
```diff
@@ -43,25 +60,43 @@ local function update()
...
+        utils.set_counter('jit_trace_abort_total', 'Overall number of abort traces',
+            lj_metrics.jit_trace_abort, nil, LJ_PREFIX)
     collectors_list.jit_mcode_size =
         utils.set_gauge('jit_mcode_size', 'Total size of all allocated machine code areas',
             lj_metrics.jit_mcode_size, nil, LJ_PREFIX)
     collectors_list.gc_steps_propagate =
         utils.set_counter('gc_steps_propagate', 'Count of incremental GC steps (propagate state)',
             lj_metrics.gc_steps_propagate, nil, LJ_PREFIX)
+    collectors_list.gc_steps_propagate_total =
+        utils.set_counter('gc_steps_propagate_total', 'Totoal count of incremental GC steps (propagate state)',
```
Comment (yngvar-antonsson) [inline metrics/tarantool/luajit.lua:92]:
```suggestion
        utils.set_counter('gc_steps_propagate_total', 'Total count of incremental GC steps (propagate state)',
```

**Review by yngvar-antonsson — CHANGES_REQUESTED** [review]

File: metrics/tarantool/luajit.lua [hunk 3]
```diff
@@ -43,25 +60,48 @@ local function update()
...
+        utils.set_counter('gc_steps_propagate_total', 'Total count of incremental GC steps (propagate state)',
+            lj_metrics.gc_steps_propagate, nil, LJ_PREFIX)
     collectors_list.gc_steps_pause =
         utils.set_counter('gc_steps_pause', 'Count of incremental GC steps (pause state)',
             lj_metrics.gc_steps_pause, nil, LJ_PREFIX)
+    collectors_list.gc_steps_pause_total =
+        utils.set_counter('gc_steps_pause_total', 'Total count of incremental GC steps (pause state)',
+            lj_metrics.gc_steps_pause, nil, LJ_PREFIX)
+
+    require('log').warn('metrics lj_gc_allocated, lj_gc_freed, lj_gc_steps_atomic, '..
```
Comment (yngvar-antonsson) [inline metrics/tarantool/luajit.lua:101]: Move it to init.lua file

**Review by yngvar-antonsson — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -14,6 +14,35 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
```
Comment (DifferentialOrange) [inline CHANGELOG.md:14]: So what's the deal? It's not like every counter should have `_total` suffix. I don't remember such requirement from promtool. Renames always bring inconveniences, but I don't see any motivation for  them here.

Comment (yngvar-antonsson) [reply CHANGELOG.md:14]: https://github.com/prometheus/client_golang/blob/618194de6ad3db637313666104533639011b470d/prometheus/testutil/promlint/promlint.go#L177

Comment (DifferentialOrange) [reply CHANGELOG.md]: Well, ok...

**Pull request merged**